.github/workflows/module-examples-tests.yaml

```diff
@@ -0,0 +1,71 @@
+name: test-examples
+
+on:
+  workflow_call:
+    secrets:
+      AUTOMATION_TOKEN:
+        required: true
+      SLACK_WEBHOOK:
+        required: true
+
+env:
+  TF_VER: '0.14.11'
+
+jobs:
+  terraform-plan:
+    if: contains(github.event.repository.name, 'aws') && (github.event_name == 'pull_request' && github.event.action == 'synchronize')
+
+    runs-on: self-hosted
+    steps:
+      - name: install python3
+        run: |
+            sudo yum update
+            sudo yum install python3
+            python3 -V
+            curl -O https://bootstrap.pypa.io/get-pip.py
+            python3 get-pip.py
+            pip3 install --upgrade awscli
+
+      - name: checkout repo code
+        uses: actions/checkout@v2
+
+      - name: setup nodejs
```

Comment:
Why do we need nodejs for?

Reply:
Nodejs is required when setting up terraform in github action using `hashicorp/setup-terraform@v2`..